cypress/e2e/Layout/Editor/layout_editor_options.cy.js

```diff
@@ -0,0 +1,337 @@
+                cy.get('.viewer-object').click();
+
+                // Wait for element to be loaded on layout
+                cy.wait('@addElement').then((interception) => {
+                    expect(interception.response.statusCode).to.eq(200);
+                });
+
+                // Check for lock status
+                cy.wait('@checkLockStatus').then((interception) => {
+                    const isLocked = interception.response.body.data[0].isLocked;
+                    expect(isLocked).to.not.be.empty;
+                    cy.log('isLocked:', isLocked);
+                });
+                
+                cy.intercept('PUT', '/layout/lock/release/*').as('unlock');
+
+                // Unlock layout
+                cy.openOptionsMenu();
+                cy.get('#unlockLayout').should('be.visible').click();
+                cy.get('button.btn-bb-unlock').click();
+
+                // Wait for the release lock request to complete
+                cy.wait('@unlock').then((interception) => {
+                    expect(interception.response.statusCode).to.equal(200);
+                });
+                
+                // Check if the user is redirected to the /layout/view page
+                cy.url().should('include', '/layout/view'); 
+
+                // Search for the layout name
+                cy.get('input[name="layout"]').clear().type(`${layoutName}{enter}`);
+                cy.get('#layouts tbody tr').should('contain.text', layoutName)
+                cy.get('#layouts tbody tr').should('have.length', 1);
+                
+                cy.openRowMenu();
+                cy.get('#layout_button_design').click();
+                cy.get('#layout-viewer').should('be.visible');
+
+                // Check for lock status
+                cy.wait('@checkLockStatus').then((interception) => {
+                    const isLocked = interception.response.body.data[0].isLocked;
+                    expect(isLocked).be.empty;
+                    cy.log('isLocked:', isLocked);
+            });
+        });
+    });
+
+    it('should enable tooltips', function() {
+        cy.wait(1000);
+        cy.openOptionsMenu();
+
+        cy.intercept('POST', '/user/pref').as('updatePreferences');
+
+        // Enable tooltips
+        // Check the current state of the tooltips checkbox
+        cy.get('#displayTooltips').then(($checkbox) => {
+            if (!$checkbox.is(':checked')) {
+                // Check the checkbox if it is currently unchecked
+                cy.wrap($checkbox).click();
+                cy.wait('@updatePreferences');
+
+                // Confirm the checkbox is checked
+                cy.get('#displayTooltips').should('be.checked');
+            } 
+        });
+
+        // Verify that tooltips are present
+        cy.get('.navbar-nav .btn-menu-option[data-toggle="tooltip"]').each(($element) => {
+            // Trigger hover to show tooltip
+            cy.wrap($element).trigger('mouseenter');
```

Comment:
Could you please change this to `mouseover` instead of `mouseenter`, because mouseenter does not reveal the tooltip?